pom.xml

```diff
@@ -13,7 +13,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
Can we change the maven.compiler.release version to 22 also to match the dockerfile build?

Reply:
Have updated to latest version